Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -4292,6 +4297,13 @@ void TestController::setServiceWorkerFetchTimeoutForTesting(double seconds)
     WKWebsiteDataStoreSetServiceWorkerFetchTimeoutForTesting(websiteDataStore(), seconds);
 }
 
+void TestController::setTracksRepaints(bool trackRepaints)
```

Comment:
This is only called with false.  We could call it resetTracksRepaints() to prevent someone in the future from using this with true when the completion handler version might be slightly more efficient.  But this is fine.